### PR TITLE
(feat infra) adding the update of AMI images with updatecli from packer-images

### PR DESCRIPTION
**Pull request by smerle33**
cannot find the issue, but the aim is to automate update of AMI from the new images from packer-images

### CODE REVIEW

File: updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    scmID: default
+
+pullrequests:
+  default:
+    kind: github
+    scmID: default
+    targets:
+      - setUbuntuAgentAMIAmd64
+      - setUbuntuAgentAMIArm64
+      - setWindowsAgentAMIamd64
```
Comment (dduportal) [inline updatecli/updatecli.d/charts/jenkins-agents-infra-release.yaml:112]:
```suggestion
      - setWindowsAgentAMIamd64
 
```

(nitpicking)

**Review by dduportal — COMMENTED** [review]
Great job!

**Comment by smerle33** [comment]
> Great job!

not working yet ... and I am stuck on this : 

`getLatestUbuntuAgentAMIAmd64
----------------------------
ERROR: RequestExpired: Request has expired.
        status code: 400, request id: d20d4e32-593b-4960-abbb-53aa24878ca6
ERROR: ✗ RequestExpired: Request has expired.
        status code: 400, request id: d20d4e32-593b-4960-abbb-53aa24878ca6
Pipeline "Bump agent templates version on jenkins-infra and jenkins-release" failed
Skipping due to:
        "sources stage:\t\"template: cfg:130:40: executing \\\"cfg\\\" at <source \\\"getLatestUbuntuAgentAMIAmd64\\\">: error calling source: parent source \\\"getLatestUbuntuAgentAMIAmd64\\\" failed\""
sources stage:  "template: cfg:130:40: executing \"cfg\" at <source \"getLatestUbuntuAgentAMIAmd64\">: error calling source: parent source \"getLatestUbuntuAgentAMIAmd64\" failed"

=============================

REPORTS:


✗ JENKINS-AGENTS-INFRA-RELEASE.YAML:
        Sources:
                ✗ [getLatestUbuntuAgentAMIAmd64] (aws/ami)
                - [getLatestUbuntuAgentAMIArm64] (aws/ami)
                - [getLatestWindowsAgentAMIAmd64] (aws/ami)
                ✔ [packerImageVersion] (githubRelease)`

**Review by dduportal — APPROVED** [review]
Looks good to me, ready to release! 

@smerle33 can you set this PR as "ready to review" so it will be merged automatically (as I've just approved it + set it as "auto-merge")?

**Review by dduportal — APPROVED** [review]

**Pull request merged**